addons/product_expiry/wizard/confirm_expiry.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+class ConfirmExpiry(models.TransientModel):
+    _name = 'expiry.picking.confirmation'
+    _description = 'Confirm Expiry'
+
+    lot_ids = fields.Many2many('stock.production.lot', readonly=True, required=True)
+    picking_id = fields.Many2one('stock.picking', readonly=True, required=True)
+    description = fields.Char('Description', compute='_compute_description')
+    show_lots = fields.Boolean('Show Lots', compute='_compute_show_lots')
```

Comment:
maybe it would be a good idea to make this wizard handle multiple pickings for a batch validation